package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "test": "mocha tests/ --reporter @digitalbazaar/mocha-w3c-interop-reporter --reporter-options abstract=\"$PWD/config/abstract.hbs\",reportDir=\"$PWD/reports\",respec=\"$PWD/config/respec.json\",suiteLog='./suite.log',templateData=\"$PWD/reports/index.json\",title=\"Data Integrity eddsa 2022 Interoperability Report 1.0\" --timeout 15000 --preserve-symlinks",
-    "lint": "eslint ."
+    "lint": "eslint --ext .js,.cjs ."
```

Comment:
Is there an eslint config file? These can go there

Reply:
this kind of surprised me, but actually I don't think there is a config file way of specifying the file extensions, but there is an eslint config file way of specifying the file glob patterns so I used that here: https://github.com/w3c/vc-di-eddsa-test-suite/pull/46/commits/40e0733bf692bed79f28badff9ef29dfa4843e75